### PR TITLE
RESOURCE-420 AWS::ApiGatewayV2::ApiMapping

### PR DESCRIPTION
**Pull request by soumyo13**
Signed-off-by: Soumyodeep Karmakar <soumyo.k13@gmail.com>

### Description

1. It contains all the singular and plural resources.
2. Added the resource_id in the singular resource.
3. Modified the docs according to the new format. 

### Issues Resolved
This is a new resource. There are no issues to be fixed.

### Check List
Please fill a box or appropriate ([x]) or mark N/A.
- [x] New functionality includes integration tests/controls
- [x] New Terraform resources
- [x] Documentation provided or updated for resources 
- [x] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

**Review by sa-progress — APPROVED** [review]

File: docs-chef-io/content/inspec/resources/aws_api_gateway_v2_api_mapping.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+end
+```
+
+## Parameters
+
+`api_mapping_id` _(required)_
+: The API mapping identifier.
+
+`domain_name` _(required)_
+: The domain name.
```
Comment (dkumaras) [inline docs-chef-io/content/inspec/resources/aws_api_gateway_v2_api_mapping.md:37]: both the parameters listed here are of string or hash type?

**Review by dkumaras — COMMENTED** [review]
Pls chk n cnfrm

File: docs-chef-io/content/inspec/resources/aws_api_gateway_v2_api_mapping.md (hunk 1 above)
Comment (soumyo13) [reply docs-chef-io/content/inspec/resources/aws_api_gateway_v2_api_mapping.md:37]: String

**Comment by soumyo13** [comment]
Correct.

**Review by dkumaras — APPROVED** [review]

**Pull request merged**